topics/cluster-spec.md

```diff
@@ -1224,8 +1224,11 @@ publish to every other node. The cluster will make sure that published
 messages are forwarded as needed.
 
 The current implementation will simply broadcast each published message
-to all other nodes, but at some point this will be optimized either
-using Bloom filters or other algorithms.
+to all other nodes.
```

Comment:
We should not use "current implementaion" to refer to the legacy pubsub, because sharded pubsub is also current implementation.

I think we need to rewrite the text above so it refers to only legacy non-sharded pubsub. Idea:

*... clients can send `SUBSCRIBE` to any node and can also send `PUBLISH` to any node. The cluster will make sure that published messages are forwarded as needed.*